lib/ansible/modules/storage/netapp/sf_volume_access_group_manager.py

```diff
@@ -0,0 +1,315 @@
+version_added: '2.3'
+author: Sumit Kumar (sumit4@netapp.com)
+description:
+- Create, destroy, or update volume access groups on SolidFire
+    - auth_basic
+
+options:
+
+    state:
+        required: true
+        description:
+        - Whether the specified volume access group should exist or not.
+        choices: ['present', 'absent']
+
+    name:
+        required: true
+        description:
+        - Name of the volume access group. It is not required to be unique, but recommended.
+
+    initiators:
+        required: false
+        type: str[]
+        description:
+        - List of initiators to include in the volume access group. If unspecified, the access group will start out without configured initiators.
+        default: None
+
+    volumes:
+        required: false
+        type: int[]
+        description:
+        - List of volumes to initially include in the volume access group. If unspecified, the access group will start without any volumes.
+        default: None
+
+    virtual_network_id:
+        required: false
+        type: int[]
+        description:
+        - The ID of the SolidFire Virtual Network ID to associate the volume access group with.
+        default: None
+
+    virtual_network_tags:
+        required: false
+        type: int[]
+        description:
+        - The ID of the VLAN Virtual Network Tag to associate the volume access group with.
+        default: None
+
+    attributes:
+        required: false
+        type: dict
+        description: List of Name/Value pairs in JSON object format.
+        default: None
+
+    volume_access_group_id:
+        required: false
+        description:
+        - The ID of the volume access group to modify or delete.
+        default: None
+
+'''
+
+EXAMPLES = """
+   - name: Create Volume Access Group
+     sf_volume_access_group_manager:
+       hostname: "{{ solidfire_hostname }}"
+       username: "{{ solidfire_username }}"
+       password: "{{ solidfire_password }}"
+       state: present
+       name: AnsibleVolumeAccessGroup
+       volumes: 7,8
+
+   - name: Modify Volume Access Group
+     sf_volume_access_group_manager:
+       hostname: "{{ solidfire_hostname }}"
+       username: "{{ solidfire_username }}"
+       password: "{{ solidfire_password }}"
+       state: present
+       volume_access_group_id: 1
+       name: AnsibleVolumeAccessGroup-Renamed
+
+   - name: Delete Volume Access Group
+     sf_volume_access_group_manager:
+       hostname: "{{ solidfire_hostname }}"
+       username: "{{ solidfire_username }}"
+       password: "{{ solidfire_password }}"
+       state: absent
+       volume_access_group_id: 1
+"""
+
+RETURN = """
+msg:
+    description: Successful creation of Volume Access Group
+    returned: success
+    type: string
+    sample: '{"changed": true, "key": value}'
+
+msg:
+    description: Successful update of Volume Access Group
+    returned: success
+    type: string
+    sample: '{"changed": true}'
+
+msg:
```

Comment:
changed: true is a common return, no need to document that. but you have `msg` several times, which does not make much sense. if the module does not return anything other than the common returns, just leave it empty